server/web/src/components/dashboard/latency-harness-panel.tsx

```
@@ -0,0 +1,770 @@
+
+  const activeRuns = runs.filter(r => r.status === 'running');
+  const connectedClients = clients.filter(c => c.isConnected);
+
+  return (
+    <div className="space-y-6">
+      {/* Error Alert */}
+      {error && (
+        <div className="p-4 rounded-xl bg-red-500/10 border border-red-500/20 text-red-400 flex items-center gap-3">
+          <AlertTriangle className="w-5 h-5 flex-shrink-0" />
+          <span>{error}</span>
+          <button onClick={() => setError(null)} className="ml-auto text-red-300 hover:text-red-200">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)